sdk/src/test/java/io/dapr/client/CloudEventTest.java

```diff
@@ -149,4 +149,26 @@ public void deserializeBytes() throws Exception {
     assertEquals("AQI=", cloudEvent.getData());
     assertArrayEquals(expected, OBJECT_MAPPER.convertValue(cloudEvent.getData(), byte[].class));
   }
+
```

Comment:
with the change for bytes as data_base64 line 142 should never happen right? can the test be removed?

Reply:
Actually, it can happen. As long as the application sends data as JSON. It is perfectly fine. But app will need to deserialize the string on the way back.

Reply:
ok good to know.